index.js

```diff
@@ -37,7 +37,7 @@ module.exports = (Hapi, options, allDone) => {
     },
     server: ['config', (done, result) => {
       const config = result.config;
-      const serverConfig = _.cloneDeep(config.server || {});
```

Comment:
I think we want to keep just cloning the server config because the engine gets set.  Basically whenever we have a `require()` we need to do a clone of that so that the config object doesn't get that stuff